### PR TITLE
SC-2612: Add "images" word in the description of the command: docker/sdk help

### PR DESCRIPTION
**Pull request by zyuzka**
Ticket: https://spryker.atlassian.net/browse/SC-2612
RG: https://release.spryker.com/release-groups/view/2295

### Bugfixes

- Adjusted command descriptions into `docker/sdk help`

### CODE REVIEW

File: generator/src/templates/deploy.bash.twig [hunk 1]
```diff
@@ -89,15 +89,15 @@ function showHelp()
...
     echo -e " ${GREEN}demo|demo-data${NC}    - Populate Spryker demo data."
     echo -e " ${GREEN}demo <section>${NC}    - Load the demo data by running the specified section from docker installation recipe for each store."
     echo -e " ${GREEN}jobs${NC}              - Generate and starts jobs."
 
     echo -e ""
     echo -e " ${GREEN}clean-data${NC}        - Remove all Spryker volumes including all storages."
-    echo -e " ${GREEN}reset${NC}             - Remove and build all Spryker containers and volumes."
-    echo -e " ${GREEN}remove|clean${NC}      - Stop and removes all Spryker containers and volumes."
+    echo -e " ${GREEN}reset${NC}             - Remove and build all Spryker images and volumes."
+    echo -e " ${GREEN}remove|clean${NC}      - Stop and removes all Spryker images and volumes."
```
Comment (sprymiker) [inline generator/src/templates/deploy.bash.twig]: Stop all Spryker containers and remove all images and volumes.

Comment (sprymiker) [reply generator/src/templates/deploy.bash.twig]: Or even rephrase completely. Ask @dmytromykhailov for the description.

Comment (dmytromykhailov) [reply generator/src/templates/deploy.bash.twig]: Let's stick to this one: Stop all Spryker containers and remove images and volumes.
I will update the Jira ticket accordingly

Comment (zyuzka) [reply generator/src/templates/deploy.bash.twig]: Updated

**Comment by alexanderM91** [comment]
Hi, @zyuzka, please prepare a release notes and release group

**Comment by alexanderM91** [comment]
Hi, @zyuzka, please prepare a release notes and release group

**Comment by zyuzka** [comment]
> Hi, @zyuzka, please prepare a release notes and release group

Hi, updated

**Pull request merged**